package.json

```diff
@@ -87,5 +88,8 @@
     "slate-react": "*",
     "source-map-loader": "^0.2.4",
     "typescript": "^3.7.2"
+  },
+  "dependencies": {
+    "react-js-to-ts": "^1.4.0"
```

Comment:
It seems `react-js-to-ts` is in both deps and devDeps, it should just be in one if at all
```suggestion
```

Also, I don't seem to see it imported directly, is it needed?